gem/ci/build.ps1

```diff
@@ -27,6 +27,19 @@ function Get-ContainerVersion
     $gitver -replace '-.*', ''
 }
 
+function Get-EnterpriseContainerVersion(
+    $Package,
+    $Token,
+    $PeVer = "2019.1.x")
+{
+    $url = "https://raw.githubusercontent.com/puppetlabs/enterprise-dist/$PeVer/packages.json"
+    $headers = @{"Authorization"="token $Token"}
+    $output = ".\packages.json"
```

Comment:
You shouldn't need to write a temporary file here or manually convert to JSON if you use `Invoke-RestMethod` instead like:

```powershell
$params = @{
  Uri = "https://raw.githubusercontent.com/puppetlabs/enterprise-dist/$PeVer/packages.json"
  Headers = @{ "Authorization" = "token $Token" }
}
$packages = Invoke-RestMethod @params
```

Reply:
@Iristyle awesome, thanks! I'll take a look!

Reply:
updated!